python/perf-kernels/tune_gemm/tune_gemm.py

```diff
@@ -16,8 +16,26 @@
 import multiprocessing
 import pandas as pd
 
-from utils.file_generator import *
-from utils.utils import *
+from utils.file_generator import (
+    gen_configStr,
+    generate_compile_driver,
+    generate_matmul_kernels,
+    generate_profile_tasks,
+    read_config,
+)
+from utils.utils import (
+    get_default_tuning_result_filename,
+    get_filename_compile_driver,
+    get_filename_myKernels,
+    get_filename_profile_driver,
+    name_to_tl_types,
+    patch_triton_compiler,
+    run_bash_command,
+    run_bash_command_wrapper,
+    tl_to_torch_types,
+    TORCH_HAS_FP8E4B8,
+    TORCH_HAS_FP8E5B16,
```

Comment:
What is the difference between importing specific functions and everything from the file?

Reply:
To be really honest, I can't answer you properly right now. It's automated `ruff` linting from Triton's `pre-commit` config, please see https://github.com/ROCm/triton/blob/triton-mlir/.pre-commit-config.yaml#L19.

`ruff` linter says `F405 <identifier> may be undefined, or defined from star imports` when we use star imports. If there's any `ruff` error the check don't pass and I'm not allowed to commit according to @micmelesse advice.

`ruff`'s justification is this one: https://docs.astral.sh/ruff/rules/undefined-local-with-import-star-usage/

If you disagree then I can try to disable this specific linter error.

Reply:
I'm just curious. And it's good to learn there is a good reason for this. Let's respect ruff in this case.

Reply:
That's fine, I'm marking this discussion as resolved.